src/app/features/dashboard-widgets/birthday-dashboard-widget/birthday-dashboard-settings.component/birthday-dashboard-settings.component.ts

```diff
@@ -43,10 +57,10 @@ export class BirthdayDashboardSettingsComponent implements OnInit {
   @Input() formControl: FormControl<BirthdayDashboardSettingsConfig>;
 
   availableEntityTypes: EntityConstructor[] = [];
-  availableProperties: string[] = [];
+  availablePropertiesMap: { [entityType: string]: string[] } = {};
+  availableEntityTypesPerRow: EntityConstructor[][] = [];
```

Comment:
The name here is not super intuitive for me. What values does this actually hold?